.eslintrc.js

```diff
@@ -6,13 +6,18 @@ module.exports = {
   },
   extends: [
     "eslint:recommended",
-    "plugin:vue/essential",
+    "plugin:vue/recommended",
     "plugin:json/recommended",
+    "prettier", // Make sure "prettier" is the last element in this list.
   ],
   parserOptions: {
     ecmaVersion: 12,
     sourceType: "module",
   },
   plugins: ["vue"],
-  rules: {},
+  rules: {
```

Comment:
I thought these rules were more annoying than helpful so I switched them off

Reply:
at first I thought this was for the script tag like `data`, `methods` which is nice to have consistent.

Yeah totes agree here

Reply:
FWIW, I would not disabled the `vue/attributes-order` rule if it is [this one](https://vuejs.org/v2/style-guide/#Element-attribute-order-recommended) as it ensures a consistent way to write attribute within templates.